package.json

```
@@ -15,6 +15,7 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc -b && vite build",
+    "prettier": "prettier --check .",
```

도구는 언제든지 바뀔 수 있는 부분이라 일반적인 커멘드 명을 추천드립니다.

```suggestion
    "format": "prettier --check .",
```

---

넵 format으로 변경하겠습니다.